ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/bridge/validate/ConflictCheckWithCommittedImpl.scala

```diff
@@ -207,4 +242,8 @@ private[validate] class ConflictCheckWithCommittedImpl(
     else
       Right(())
   }
+
+  private implicit class FutureValidationOps(val e: Future[Option[Rejection]]) {
+    def invertToEitherT: EitherT[Future, Rejection, Unit] = EitherT.fromOptionF(e, ()).swap
```

Comment:
I would probably either inline it and/or create a plain helper function and assign it's argument's expression (i.e. at the call site) to a val (bike shedding)